features/provider/admin/applications/new.feature

```diff
@@ -71,7 +71,7 @@ Feature: Audience's new application page
     # Travel in time to ensure the new products are within the first 20 results
     # (because the products are sorted by updated_at: :desc)
     Given 5 minutes pass
-    And 18 products and 1 backend apis
+    And 18 products and 0 backend apis
```

Comment:
We don't actually need any backend apis here, it's just that at first I didn't think we could pass 0 here.